metrics-dropwizard/src/main/resources/reference.conf

```diff
@@ -0,0 +1,63 @@
+usi.metrics {
```

Comment:
Shouldn't this be in `metrics` instead of `metrics-dropwizard`?

Reply:
I'd say everything in this conf is implementation specific: reporters, histogram settings (based by HdrHistogram) and even name prefix - original interface doesn't define any of those (for a good reason)

Reply:
This about `DummyMetris` used for tests that doesn't need any of these.